src/js/features/beacon/utils.ts

```diff
@@ -78,12 +78,18 @@ export const extractBeaconDiscoveryOverview = (response: BeaconQueryResponse): O
   // Bento-specific counts/chart data
   ...(response.info?.bento
     ? {
-        biosampleCount: response.info.bento.biosamples?.count,
+        biosampleCount: response?.info.bento.biosamples?.count,
```

Comment:
why this change? if response.info.bento exists, it follows that response exists.